### ISSUE TITLE
Update function does not update all ids 

### ISSUE THREAD

**Issue by Adrek** [jonataslaw/getx#1079]
**Describe the bug**
I was implementing GetX in a large project and noticed that it did not update all the GetBuilders ids that I passed in the "update" function. It only updates the first id of the GetBuilder.

So to test it, I have created another clean project and it only has one page. But the same thing happens, only the first id of the GetBuilder that I pass in the "update" function is updated.

PD: I'm newbie using GetX, before I used Flutter Bloc. 

**Reproduction code
Main.dart
```dart
import 'package:dont_update_getx/controller/global_controller.dart';
import 'package:flutter/material.dart';
import 'package:get/get.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    Get.put(GlobalController());
    return GetMaterialApp(
      title: 'Material App',
      home: Scaffold(
          appBar: AppBar(
            actions: [
              GetBuilder<GlobalController>(
                  id: 'action',
                  builder: (_) {
                    return FlatButton(
                      child: Text('AppName=> ${_.appName}'),
                      onPressed: () {},
                    );
                  })
            ],
          ),
          body: ContentWidget()),
    );
  }
}

class ContentWidget extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return GetBuilder<GlobalController>(
      id: 'content',
      builder: (_) => Column(
        children: [
          Center(
            child: Text('${_.appName}'),
          ),
          MaterialButton(
            child: Text('Change app name'),
            color: Colors.blue,
            onPressed: () {
              _.changeAppName();
            },
          )
        ],
      ),
    );
  }
}
```

global_controller.dart
```dart
import 'package:get/state_manager.dart';

class GlobalController extends GetxController {
  String _appName = 'HelloWorld';

  String get appName => this._appName;

  changeAppName() {
    print('GETX=> Changing Name');
    this._appName = 'How are you?';
    update(['content', 'action']);
  }
}
```

**To Reproduce**
1. Clone https://github.com/Adrek/dont_update_getx

**Expected behavior**
```dart
changeAppName() {
    print('GETX=> Changing Name');
    this._appName = 'How are you?';
    update(['content', 'action']);
  }
```
>>> 
They should update the GetBuilders with id 'content' and 'action', **but only the first id 'content' is being updated.**

I tried changing the order of the id >> update (['action', 'content']) << and in that case only the GetBuild with 'action' is updated, but 'content' is not updated. 

**Screenshots**
1- https://i.imgur.com/xKwD8Wm.png
2- https://i.imgur.com/icy5IKd.png

**Flutter Version:**
Flutter 1.22.6 • channel stable • https://github.com/flutter/flutter.git 
Framework • revision 9b2d32b605 (3 weeks ago) • 2021-01-22 14:36:39 -0800
Engine • revision 2f0af37152
Tools • Dart 2.10.5

**Getx Version:**
get: ^3.25.4

**Describe on which device you found the bug:**
Android Studio emulator

**Minimal reproduce code**
https://github.com/Adrek/dont_update_getx

Thank you very much for your help

**Comment by ghprod** [comment]
Hi

Any reason why you don't use `Rx` and put dependency with `Binding`?

**Comment by Adrek** [comment]
> Hi
> 
> Any reason why you don't use `Rx` and put dependency with `Binding`?

Hi ghprod, there isn't. I have tested it with Rx Obs and if it works. So if there is no other way I will use Rx

Even so, I am left with that question: What is the reason why the update of all the id does not work? In theory it should work but it doesn't. Is Update deprecated? Will no longer be supported on that?
 
Thanks for your answer.
Greetings.

PS: In the case of using Obs, How can I add a type (String, int, bool) to a variable.

**Comment by jonataslaw** [comment]
Thanks for providing a reproduction code, if I can confirm the behavior, it will be fixed in the next version.

**Comment by davey06** [comment]
> PS: In the case of using Obs, How can I add a type (String, int, bool) to a variable.
did you mean getting the type from RxType? if yes, Rx<T> has `.value` method to extract the type.

example: RxString foo;

**Comment by eduardoflorence** [comment]
Fix in 3.25.6
Update GetX

**Comment by Adrek** [comment]
Thanks. now it works. This package is awesome.
